api/src/test/java/org/openmrs/module/appointments/service/impl/AppointmentServiceDefinitionServiceImplTest.java

```diff
@@ -26,6 +27,7 @@
 import static org.mockito.Mockito.when;
 import static org.powermock.api.mockito.PowerMockito.mockStatic;
 
+@PowerMockIgnore("javax.management.*")
```

Comment:
why is this needed?

Reply:
PowerMockito initiates loader with classes found in this library ("javax.management.*"), so we face a linkage error when running it on the current class. 
`2022-06-10 10:41:01,526 main ERROR Could not reconfigure JMX java.lang.LinkageError: loader constraint violation: loader (instance of org/powermock/core/classloader/javassist/JavassistMockClassLoader) previously initiated loading for a different type with name "javax/management/MBeanServer"
`  

Found the solution from this link: [Mockito + PowerMock LinkageError while mocking system class](https://stackoverflow.com/questions/16520699/mockito-powermock-linkageerror-while-mocking-system-class)